Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,221 @@
+                dateobj = dateobj - timedelta(days=int(number)*365)
+        except Exception:
+            return_error('invalid dateformat supplied')
+        return dateobj
+
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            last_run['lt_time'] = parse_date_range(FETCH_TIME)
+        else:
+            last_run['lt_time']= datetime.datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=get_incident_id(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                serialized_obj = helpers.serialize_object(inc[0])
+                rawjson = json.dumps(serialized_obj,default = myconverter)
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
+                    'rawJSON': rawjson,
+                    'severity' : SEVERITY[inc[0]['incident']['severity']]
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
+                    demisto.setLastRun({'lt_time':inc[0]['incident']['incidentCreationDate'].strftime("%d/%m/%Y %H:%M:%S.%f")})
+        demisto.incidents(incidents)
+
+
+    def updateIncidents():
+        args = demisto.args()
+        inc_id = args['incidentId']
+        args.pop('incidentId')
+        if args['status'] == 'New':
+            args['status'] = 'incident.status.New'
+        return client.service.updateIncidents(updateBatch={'batchId':str(random.randint(11111,99999)),'incidentId':inc_id,'incidentAttributes':args})
+
+
+
+    if demisto.command() == 'fetch-incidents':
+        fetchincidents()
+        
+    elif demisto.command() =='test-module':
+        try:
+            inc_list = get_incident_id(datetime.datetime.now())
+            demisto.results("ok")
+        except:
+            return_error("Invalid instance details")
+            
+    elif demisto.command() == 'dlp-incident-details':
+        inc_details = incidentDetail( demisto.args()['incidentId'])
+        res = helpers.serialize_object(inc_details[0])
+        json_res = json.loads(json.dumps(res,default = myconverter))
+        demisto.results({'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents':json_res,
+        'EntryContext':{'Symantec':{"DLP":{'Incident':json_res}}}
+        })
+        
+    elif demisto.command() == 'dlp-update-incident':
+        demisto.results(updateIncidents())
+
+  type: python
+  commands:
+  - name: dlp-incident-details
+    arguments:
+    - name: incidentId
+      required: true
+      description: incident Id  from DLP console to fetch the data of
+      outputs:
```

Comment:
you have 1 extra tab / 2 spaces here.
move `outputs` back 2 spaces to align at the same level as arguments